amy/scripts/seed_training_requirements.py

```diff
@@ -9,7 +9,14 @@
 # If an entry needs to be removed from the database, remove it from e.g.
 # `EMAIL_TEMPLATES`, and put its' ID in `DEPRECATED_EMAIL_TEMPLATES`.
 
-DEPRECATED_TRAINING_REQUIREMENTS: list[str] = []
+DEPRECATED_TRAINING_REQUIREMENTS: list[str] = [
+    "DC Homework",
+    "SWC Homework",
+    "LC Homework",
+    "DC Demo",
+    "SWC Demo",
+    "LC Demo",
+]
```

Comment:
Just want to make sure: were these migrated in production? I'm not sure what would happen if these were removed for existing users.

Reply:
Okay, I saw a migration removing these entries. We should not keep 2 systems for managing "static" (seeded) objects. I would suggest sticking to the seeding script, unless a data migration is actually required.

Reply:
Okay, I can remove that step from the migration. This is one thing I am uncertain of - are the seeding scripts run when deploying new versions to production?